fab/fabfile.py

```diff
@@ -165,7 +201,8 @@ def _setup_path():
     env.log_dir = posixpath.join(env.home, 'www', env.environment, 'log')
     env.releases = posixpath.join(env.root, 'releases')
     env.code_current = posixpath.join(env.root, 'current')
-    env.code_root = posixpath.join(env.releases, time.strftime('%Y-%m-%d_%H.%M', time.gmtime(time.time())))
+    timestamp = time.strftime('%Y-%m-%d_%H.%M', time.gmtime(time.time()))
```

Comment:
Does it matter if this time stamp is different from the one used in `CodeToDeploy`? Would it make sense to use `code_to_deploy.deploy_time` instead?